### ISSUE TITLE
Adressauflösung defekt

### ISSUE THREAD

**Issue by adlerweb** [FreifunkMYK/PW-Freifunk-Starter#61]
Aktuell werden alle Knoten als "no address found" angezeigt

**Comment by kreativmonkey** [comment]
Ja, das Problem flutet mir derzeit auch mein error Log!!! Handelt sich um folgenden [Code](https://github.com/FreifunkMYK/PW-Freifunk-Starter/blob/master/modules/ffNodeInfo.module#L81:L100). Scheint evtl. Probleme mit der API Nutzung zu sein oder die API ist umgestellt worden.... Eigentlich sollten nur die Nodes geprüft werden die keine Adresse haben damit wir nicht zu viele Anfragen an die API stellen, nun sendet er natürlich für alle.... das müsste man evtl. anders lösen.
